### ISSUE TITLE
Hotgraphic popups - Disabled paging buttons are not focusable

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-hotgraphic#241]
### Subject of the issue/enhancement/features
In Hotgraphic popups, disabled paging buttons are not focusable via keyboard navigation.  This can lead to a lesser user experience as the buttons are completely ignored by screenreaders.

Suggest removing `visibility: hidden` for disabled paging buttons.

Requires https://github.com/adaptlearning/adapt-contrib-core/issues/220 which removes the tabindex attribute.

### Your environment
* Framework 5.x

### Steps to reproduce
Add a Hotgraphic component to a course. Click on an item to open the Notify popup. Note that the paging buttons call a11y.toggleAccessibleEnabled to enable or disable the buttons. That function then calls a11y.toggleEnabled.

### Expected behaviour
Disabled buttons should be visible, focusable and marked as disabled.

### Actual behaviour
Disabled buttons are hidden and not focusable.
